### PR TITLE
pkg: add gatewayport and leafnodeport to service

### PR DESCRIPTION
**Pull request by squat**
This commit ensures that the gateway and leafnode ports are added to the
management service created for a cluster.

Signed-off-by: Lucas Servén Marín <lserven@gmail.com>

Plz let me know if these ports should also be added to the client service.
The ports were added to the headless service to match the Helm charts
and raw YAML files, e.g. 
https://github.com/nats-io/k8s/blob/master/helm/charts/nats/templates/service.yaml#L35-L38
Similarly, the port names are pluralized, i.e. `leafnodes` and `gateways`
to match.

### CODE REVIEW

File: pkg/util/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -138,6 +138,8 @@ func CreateMgmtService(
 	clusterName, clusterVersion, ns string,
 	owner metav1.OwnerReference,
 	websocketPort int,
+	gatewayPort int,
+	leafenodePort int,
```
Comment (wallyqs) [inline pkg/util/kubernetes/kubernetes.go:142]:
```suggestion
	leafnodePort int,
```

**Comment by wallyqs** [comment]
I think just adding to the management service is fine, thanks for the contribution!

File: pkg/util/kubernetes/kubernetes.go (hunk 1 above)
Comment (squat) [reply pkg/util/kubernetes/kubernetes.go]: :+1:

**Comment by squat** [comment]
@wallyqs do we need to do anything special to get the CI to finish?

**Comment by wallyqs** [comment]
Not sure what changed in travis, the build itself seems fine: https://travis-ci.com/github/nats-io/nats-operator/builds/223332812

**Pull request merged**